.github/actions/build-extra-tests/action.yml

```diff
@@ -0,0 +1,28 @@
+name: build-extra-tests
+description: 'Builds extra test required for some flows'
+
+inputs:
+  tools-version:
+    description: Which toolchain to build
+    required: false
+    default: 'riscv-tools'
+  cache-key:
+    description: Use this for caching
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/cache@v2
+      id: build-extra-tools-cache
+      with:
+        path: extra-tests-install
+        key: ${{ needs.make-keys.outputs.extra-tests-cache-key }}
+        restore-keys: ${{ needs.make-keys.outputs.extra-tests-cache-key }}
+    - name: Build extra tests if not cached
+      run: |
+        export RISCV="/__w/chipyard/chipyard/riscv-tools-install"
+        export LD_LIBRARY_PATH="$RISCV/lib"
+        export PATH="$RISCV/bin:$PATH"
```

Comment:
I'm not sure if this should be deleted or not. My impression is that yes you can delete this but I would double check.